src/java.base/share/man/java.md

```diff
@@ -2883,6 +2883,48 @@ Java HotSpot VM.
     expensive operations. Using a lower value will cause heap memory to be
     uncommitted earlier, at the risk of soon having to commit it again.
 
+`-XX:+UseShenandoahGC`
+:   Enables the use of the Shenandoah garbage collector. This is a low pause
+    time, concurrent garbage collector. Its pause times are not proportional to
+    the size of the heap. Shenandoah garbage collector can work with compressed
+    pointers. See `-XX:UseCompressedOops` for further information about
+    compressed oops.
+
+`-XX:ShenandoahGCMode=`*mode*
+:   Sets the GC mode for Shenandoah GC to use. By default, this option is set
+    to `satb`. Among other things, this defines which barriers are in use.
+    Possible mode values include the following:
+
+    `satb`
+    :   Snapshot-at-the-beginning concurrent GC (three pass mark-evac-update).
```

Comment:
This is a long-standing problem with the name. It kinda implies `generational` is not `satb`. It actually is. Maybe we should rename `satb` to `single` or something. A lot of bikeshedding would ensue.

Reply:
I agree. For non-experimental options, we can follow the established deprecate & rename protocol.

May be we could, in the inerim, add verbage in the description to avoid the confusion in the current naming as described by Aleksey.